lib/db.js

```diff
@@ -57,16 +57,21 @@ function mongoConnectionStringOS3(cb){
 }
 
 function getMongoConnectionUrl(cb){
+  if (process.env.FH_USE_LOCAL_DB) {
+    // get the local connection URL here
+    return cb(null, 'dummy');
+  }
   if(process.env.FH_MONGODB_CONN_URL){
     return cb(undefined, process.env.FH_MONGODB_CONN_URL);
-  } else if (process.env.OPENSHIFT_MONGODB_DB_HOST) {
+  }
+  if (process.env.OPENSHIFT_MONGODB_DB_HOST) {
     return mongoConnectionStringOS2(cb);
-  } else if ("openshift3" === process.env.FH_MBAAS_TYPE){
+  } 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)